src/goto-programs/goto_convert.cpp

```diff
@@ -498,7 +502,7 @@ bool goto_convertt::rewrite_vla_decl_size(exprt &size, goto_programt &dest)
     return true;
   }
 
-  // A constant array
+  // This should never happen, this is a constant temporary symbol.
```

Comment:
I don't understand this comment: temporary symbols would be caught in the `if`, because they are named `tmp$`, right?

Reply:
The opposite, if `find` is equal to `npos` it means that the element was not found.